packages/flutter_tools/lib/src/windows/migrations/version_migration.dart

```diff
@@ -0,0 +1,97 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import '../../base/file_system.dart';
+import '../../base/project_migrator.dart';
+import '../../cmake_project.dart';
+
+const String _cmakeFileBefore =
```

Comment:
Would it be preferable to put these long hardcoded strings into their own file(s)?

Reply:
Prior art also hardcodes strings inside the migration itself: https://github.com/flutter/flutter/blob/2379b18f8db2fae3bc8f189c7c24e98f6fdff9aa/packages/flutter_tools/lib/src/migrations/xcode_thin_binary_build_phase_input_paths_migration.dart#L41-L62

I can move this to another file if folks feel strongly about this.

Reply:
I weakly would prefer it inlined to make it easier to grep for a string and then see the usage within the tool.